### PR TITLE
Streamline usage of quoted message view.

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
A few fixes that improve perf _and_ fix the "quoted text is fuzzy" issue.

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m [hunk 1]
```diff
@@ -101,6 +107,8 @@ - (UIColor *)highlightColor
 
 - (void)createContents
 {
+    OWSAssert(!self.boundsStrokeView);
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m]: A comment could clarify why we're doing this: `// ensure called only once`

File: Signal/src/views/QuotedReplyPreview.swift [hunk 2]
```diff
@@ -36,7 +36,6 @@ class QuotedReplyPreview: UIView {
 
     func updateContents() {
         subviews.forEach { $0.removeFromSuperview() }
-        self.quotedMessageView = OWSQuotedMessageView(forPreview: quotedReply)
```
[the remark this answers is not in the document]
Comment (michaelkirk-signal) [reply Signal/src/views/QuotedReplyPreview.swift]: `updateContents` is only called in two situations:

1. when we init
2. when contentSizeCategoryDidChange

(1.) is redundant, but for (2.) we need this to stay, or the bar won't reflect the new height of the QuotedMessageView.

We could revert this change and get rid of the redundancy by removing the other `OWSQuotedMessageView()` on line 26, but then we have to make `quotedMessageView` an optional and deal with any logical ramifications - I think it just means a new `guard` in `updateHeight`.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m (hunk 1 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m]: Done.

File: Signal/src/views/QuotedReplyPreview.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (charlesmchen-signal) [reply Signal/src/views/QuotedReplyPreview.swift]: Done.

img_0094
img_0093
img_0092 1

**Pull request merged**